utils/spdx/src/test/kotlin/model/SpdxDocumentTest.kt

```diff
@@ -33,10 +33,10 @@ import org.ossreviewtoolkit.utils.spdx.SpdxModelMapper
 import org.ossreviewtoolkit.utils.spdx.yamlMapper
 
 /**
- * This test uses the following test assets copied from the SPDX 2.2.1 specification examples.
+ * This test uses the following test assets copied from the SPDX 2.2.2 specification examples.
  *
- * 1. https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXYAMLExample-2.2.spdx.yaml
- * 2. https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXJSONExample-v2.2.spdx.json
+ * 1. https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXYAMLExample-2.2.spdx.yaml
+ * 2. https://github.com/spdx/spdx-spec/blob/development/v2.2.2/examples/SPDXJSONExample-v2.2.spdx.json
```

Comment:
Have you checked whether these files are identical?

Reply:
I believe I did check when I was originally doing these (old) changes, but I just did again: On https://github.com/spdx/spdx-spec/compare/v2.2.1...v2.2.2 the "Files changes" tab does not list these files, so there are no changes.